site/package.json

```diff
@@ -10,6 +10,7 @@
     "check:all": "yarn format:check && yarn lint && yarn test",
     "chromatic": "chromatic",
     "dev": "webpack-dev-server --config=webpack.dev.ts",
+    "dev:vite": "vite",
```

Comment:
I like that you're leaving webpack in and encouraging folks to try vite. Seems like the right move and then when we're confident Vite works, we can remove webpack 👍🏼